attributes/cacher.rb

```diff
@@ -0,0 +1,2 @@
+default['ssh_known_hosts']['cacher']['data_bag'] = 'server_data'
+default['ssh_known_hosts']['cacher']['data_bag_item'] = 'known_hosts'
```

Comment:
Well from chef's pow, yes, it's pointless, but from the developer's point of view clear separation of concerns be a good thing.